charts/incubator/openspeedtest/values.yaml

```diff
@@ -0,0 +1,30 @@
+image:
+  repository: tccr.io/truecharts/openspeedtest
+  pullPolicy: IfNotPresent
+  tag: v0.20.1216@sha256:b18c87da9d022c81edcec0cfa21d32669eb99a5e0418a520569ad70e3de939d8
```

Comment:
For now change it to
repository: openspeedtest/latest
tag: latest@sha256:ac0f1b4313bab30fd33b07addacfd3cbf51ed8722225c7514a31096433245952
So we can see some tests running.

And later when the image is mirrored in our repo, we can change it back